plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -384,7 +341,12 @@ static int is_timestamped_fleet_config_path(struct flb_in_calyptia_fleet_config
         return FLB_FALSE;
     }
 
-    if (strcmp(end, ".conf") == 0) {
+    if (ctx->fleet_config_legacy_format) {
```

Comment:
not sure if relevant but when a config_format context is created, there is a flag called `format` that indicates if the config content is `FLB_CF_CLASSIC` or `FLB_CF_YAML`, ref: